### PR TITLE
optimize ParseIP

### PR DESCRIPTION
**Pull request by helen-frank**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
optimize ParseIP
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/karmadactl/cmdinit/utils/ip.go [hunk 1]
```diff
@@ -5,19 +5,14 @@ import (
 	"net"
 )
 
-//ParseIP parse an ip address and return whether it is a v4 or v6 ip address
+// ParseIP parse an ip address and return whether it is a v4 or v6 ip address
```
Comment (RainbowMango) [inline pkg/karmadactl/cmdinit/utils/ip.go:8]:
```suggestion
// ParseIP parses an IP address and returns whether it is a v4 or v6 IP address
```

Comment (helen-frank) [reply pkg/karmadactl/cmdinit/utils/ip.go:8]: ok

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Comment by XiShanYongYe-Chang** [comment]
Change the label to the correct one.
/kind cleanup
/remove-kind feature

**Pull request merged**